dabuild.in

```diff
@@ -73,6 +73,26 @@ ABUILD_VOLUMES="-v ${HOME}/.abuild:/home/builder/.abuild \
   -v ${PWD%/aports/*}/aports:/home/builder/aports \
   -v ${ABUILD_PACKAGES}:/home/builder/packages"
 
+## if we have ~/.ccache on host, bind-mount it; else use named per-arch volumes
+if [ -w "$HOME/.ccache" ]; then
```

Comment:
Why should we support bind mounting ccache directory? Are there use cases where you want to share ccache? or does it make sense to have direct access to it? It will also start using my local ccache directory if it existed even if I don't want it to.